recipes/pandas-summary/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pandas-summary" %}
+{% set version = "0.0.4" %}
+{% set md5 = "12e245c599686ce270fe233e4996c3c4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pandas
```

Comment:
Can we please drop `numpy` and `pandas` from the build dependencies? Unless of course we need them at build for some reason.